pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -363,7 +483,7 @@ func (o *MustGatherOptions) newPod(node string) *corev1.Pod {
 			Containers: []corev1.Container{
 				{
 					Name:    "copy",
-					Image:   o.Image,
+					Image:   o.Images[0],
```

Comment:
Why always `o.Images[0]` ? I know it doesn't matter, it should not but why not using the same as for init?

Reply:
Done